browser/ui/views/frame/split_view/brave_multi_contents_view.cc

```diff
@@ -72,15 +69,20 @@ void BraveMultiContentsView::UpdateContentsBorder() {
         } else {
           contents_container_view->SetBorder(views::CreateBorderPainter(
               views::Painter::CreateRoundRectWith1PxBorderPainter(
-                  cp->GetColor(kColorBraveSplitViewInactiveWebViewBorder),
-                  cp->GetColor(kColorToolbar), corner_radius, SkBlendMode::kSrc,
+                  GetColorProvider()->GetColor(
+                      kColorBraveSplitViewInactiveWebViewBorder),
+                  GetColorProvider()->GetColor(kColorToolbar), corner_radius,
+                  SkBlendMode::kSrc,
                   /*anti_alias*/ true,
                   /*should_border_scale*/ true),
               gfx::Insets(kBorderThickness)));
         }
+        // Mini toolbar should only be visible for the inactive contents
+        // container view.
+        contents_container_view->GetMiniToolbar()->SetVisible(!is_active);
```

Comment:
We don't need to show chromium's mini toolbar. Pushed fixup - https://github.com/brave/brave-core/pull/28944/commits/82afd117d65db0c90d576ca4010598357113035e
cc @cdesouza-chromium @emerick